java/operator/src/main/resources/com/instaclustr/test-dc.yaml

```diff
@@ -4,13 +4,17 @@ metadata:
   name: test-dc
 spec:
   replicas: 6
-  image: instaclustr/k8s-cassandra
+  version: 3.11.2
+  image: instaclustr/k8s-cassandra:3.11.2
```

Comment:
can we change this to the image name/tag used by default with the build scripts and stored in gcr.io ?

Reply:
Yep, will change that. I'll also remove `version` -- that was me experimenting with things, but I eventually realised that its just far easier to allow a full image string and just have a big fat warning that the operator only works with C* images built with the operator in mind.